fixed-requirements.txt

```diff
@@ -41,7 +41,7 @@ six==1.13.0
 # NOTE: sseclient has various issues which sometimes hang the connection for a long time, etc.
 sseclient-py==1.7
 stevedore==1.30.1
-tooz==1.66.1
+tooz==2.8.0; python_version >= '3.6'
```

Comment:
Since py2 removal, we’re on py3.6 anyways.
so feel free to remove the condition part.

Reply:
Same idea here. Already pushed it before I saw your comments.